src/providers/historical-telemetry-provider.js

```diff
@@ -84,7 +98,24 @@ export default class YamcsHistoricalTelemetryProvider {
             .then(res => {return this.convertSampleHistory(id, res);});
     }
 
+    convertEventHistory(id, results) {
+        if (!results.event) {
+            return [];
+        }
+
+        return results.event.map(e => {
+            return {
+                id,
+                ...e
+            };
+        }) || [];
```

Comment:
why do we need an || here? Wouldn't a map on an empty array return an empty array?